test/run

```diff
@@ -8,6 +8,12 @@ TEST_SCENARIO="${TEST_SCENARIO:-}"
 [ "${TEST_SCENARIO}" = "${TEST_SCENARIO##firefox}" ] || export TEST_BROWSER=firefox
 export RUN_TESTS_OPTIONS=--track-naughties
 
+TEST_SCENARIO=${TEST_SCENARIO:-}
+
```

Comment:
This line is redundant and confusing, line 7 already does that.

Reply:
Woops, fixed and added the issue to the commit.